src/components/Button.js

```diff
@@ -1,14 +1,26 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 
-const Button = ({ name }) => (
-  <>
-    <div>
-      {name}
-    </div>
-  </>
-);
-
-Button.propTypes = PropTypes.string.isRequired;
+const Button = ({ name, clickHandler }) => {
+  const handleClick3 = btnName => clickHandler(btnName);
+
+  return (
+    <>
+      <button
+        className="calculator-button"
+        type="button"
+        onClick={() => handleClick3(name)}
+        tabIndex={name}
```

Comment:
- [ ] you don't need to make a function here, simply pass `clickHandler` directly to the button.